app/scripts/components/common/chart/constant.js

```diff
@@ -0,0 +1,13 @@
+export const chartHeight = '32rem';
```

Comment:
Should we consider the chart's height as a ratio of its width? Something to think about

Reply:
Yeah that is a good idea especially we will have some smaller charts for analysis page. I will do follow up about it.